hbase-protocol-shaded/src/main/protobuf/server/master/Master.proto

```diff
@@ -696,6 +696,7 @@ message ListDecommissionedRegionServersResponse {
 message DecommissionRegionServersRequest {
   repeated ServerName server_name = 1;
   required bool offload = 2;
+  optional bool match_host_name_only = 3 [default = false];
```

Comment:
I don't think that we want to prescribe a default value in the protobuf. Instead, the service should interpret the absence of a value appropriately.